cachi2/core/scm.py

```diff
@@ -120,8 +120,6 @@ def clone_as_tarball(url: str, ref: str, to_path: Path) -> None:
             _reset_git_head(repo, ref)
 
             with tarfile.open(to_path, mode="w:gz") as archive:
-                # GitPython wrongly annotates working_dir as Optional, it cannot be None
-                assert repo.working_dir is not None  # nosec assert_used
```

Comment:
Is this not needed anymore? Since you haven't disabled a rule, does bandit not report a problem anymore?

Reply:
No, `working_tree_dir` attribute is Optional, not `working_dir`.
https://github.com/gitpython-developers/GitPython/blob/main/git/repo/base.py#L126